app/main.py

```diff
@@ -1,8 +1,32 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.__class__.people[name] = self
+
+    @classmethod
```

Comment:
You don't really need classmethod here. You can set attributes of wife and husband simply by accessing Person.people["name"].wife | husband